builtin/providers/aws/resource_aws_route_table.go

```diff
@@ -148,6 +148,10 @@ func resourceAwsRouteTableRead(d *schema.ResourceData, meta interface{}) error {
 			continue
 		}
 
+		if r.DestinationPrefixListID != nil {
+			continue
+		}
```

Comment:
just a nit - we could use some explanatory comments on why we skip these routes.